### PR TITLE
polybar: include example config

### LINKED ISSUE #27464: Latest Polybar update removes example files on post_install
Latest Polybar update removes example files on post_install

Affects polybar when installing with xbps-install -S polybar and building with xbps-src

### Expected behavior
Being able to copy the example config from /usr/share/doc/polybar

### Actual behavior
removes this folder on post_install:
rm -rf ${DESTDIR}/usr/share/doc/polybar

removing this line fixes this issue

### Steps to reproduce the behavior
xbps-install -S polybar
or building with xbps-src

### PR DESCRIPTION
**Pull request by pbui**
Fixes #27464

### CODE REVIEW

**Comment by pbui** [comment]
@ericonr @fahrberg This should remove everything *except* the sample config from `/usr/share/doc/polybar`.

File: srcpkgs/polybar/template [hunk 1]
```diff
@@ -51,6 +51,6 @@ if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
 fi
 
 post_install() {
-	rm -rf ${DESTDIR}/usr/share/doc/polybar
+	find ${DESTDIR}/usr/share/doc/polybar/ -mindepth 1 ! -name config -delete
```
Comment (sgn) [inline srcpkgs/polybar/template:54]: IMO, it's better to remove everything is `/usr/share/doc/polybar` and `vdoc config` or `vsconf config`.

Comment (ericonr) [reply srcpkgs/polybar/template:54]: The only problem with this approach is that it breaks if upstream changes where they put the example config. That isn't the case here.

Comment (pbui) [reply srcpkgs/polybar/template]: OK, I've modifed it to `rm` the directory and then `vdoc` the `config` file.  Here is the output of `xls`:

```
/usr/bin/polybar
/usr/bin/polybar-msg
/usr/share/bash-completion/completions/polybar
/usr/share/doc/polybar/config
/usr/share/licenses/polybar/LICENSE
/usr/share/man/man1/polybar.1
/usr/share/man/man5/polybar.5
/usr/share/zsh/site-functions/_polybar
/usr/share/zsh/site-functions/_polybar_msg
```

**Comment by ericonr** [comment]
We might need to watch out for changes in the config location, but I hope that's unlikely to happen.

**Pull request merged**